README.md

```diff
@@ -1,57 +1,35 @@
-# p-report-template
+# p-report
+[![Build Status](https://travis-ci.org/jsk-report-template/p-report.svg?branch=master)](https://travis-ci.org/jsk-report-template/p-report)
 
 Latex template for p-report
 
-[![Build Status](https://travis-ci.org/furushchev/p-report-template.svg)](https://travis-ci.org/furushchev/p-report-template)
-
 ### 1. Prerequisities
 
 ```bash
-# only for ubuntu 12.04
-$ sudo apt-add-repository ppa:texlive-backports/ppa
-$ sudo apt-get update
-$ sudo apt-get install lsb-release
+sudo apt-get update
+make install
```

Comment:
Previously, `sudo apt-get update` is required because we need this after `apt-add-repository`.
But in the new policy, we don't write `apt-add-repository`, so `sudo apt-get update` is not required.
And `make install` is executed automatically in `make`, so I think this is not also required.
So, in my opinion, `Prerequisities` should be removed if we choose the new policy.

Reply:
hmm, from the viewpoint of maintenance, we don't have presice test, so we cannot support it anymore, I think.